src/app/theme.ts

```diff
@@ -0,0 +1,6 @@
+
+import { createTheme } from '@mui/material/styles';
```

Comment:
Why don't you define the theme's values?

You can check this link for reference: https://www.welcomedeveloper.com/posts/react-mui-theme/#:~:text=Then%2C%20finally%20the%20part%20we%20developers%20love%20most!%20Let%E2%80%99s%20write%20some%20code%20to%20create%20a%20custom%20theme%20for%20our%20React%20app%3A